charts/datadog/values.yaml

```diff
@@ -1061,6 +1061,56 @@ clusterAgent:
     # clusterAgent.admissionController.port -- Set port of cluster-agent admission controller service
     port: 8000
 
+    agentSidecarInjection:
+      # clusterAgent.admissionController.agentSidecarInjection.enabled -- Enables Datadog Agent sidecar injection.
+
+      ## When enabled, Admission Controller mutating webhook will inject Agent sidecar with minimal configuration in every pods meeting configured criteria.
+      ## ref: https://docs.datadoghq.com/integrations/eks_fargate
```

Comment:
do you think this ref is needed here? maybe we can put it a few lines below where "fargate" is mentioned

Reply:
yeah, agree. I'll move where provider is defined.